terraform/modules/storage/database/mongo/locals.tf

```diff
@@ -1,3 +1,3 @@
 locals {
-    test-cmd = "exec test $$( mongosh --quiet --eval 'rs.status().ok' ) -eq 1"
+  test-cmd = "exec test $$( mongosh --quiet --eval 'rs.status().ok' ) -eq 1"
 }
```

Comment:
Canonical format should include a an empty line at the end of the file

Reply:
`terraform fmt` does not add it

Reply:
I know, but I think it should still be part of _our_ conventions (like LF line returns). Maybe we should apply dos2unix on all files.

Reply:
So we also need a way to ensure it. The purpose of this formatting is to validate terraform formatting in https://github.com/aneoconsulting/ArmoniK.Core/pull/287

Reply:
I would say, even though it is not exactly required, it can be nice to take the opportunity of this PR to run at least dos2unix on all the text files, because we know that's something we will want anyway in the future. We could then investigate those tools in another PR.

```sh
find . -type f | xargs dos2unix
```

Reply:
We'll need to exclude .git files...

Reply:
I tried on terraform files and they are already unix files

Reply:
We could add more check in the formatting pipeline